Example/Podfile

```diff
@@ -6,7 +6,7 @@ target 'YelpAPI_Example', :exclusive => true do
   pod "YelpAPI", :path => "../"
 end
 
-target 'YelpAPI_Tests', :exclusive => true do
+target 'YelpAPI_PodsTests', :exclusive => true do
   pod 'TDOAuth', '~> 1.1'
```

Comment:
Can this line be removed? I think it's superfluous since YelpAPI lists it as a dependency.